src/components/EffectiveLocationFacet/EffectiveLocationFacet.js

```diff
@@ -0,0 +1,88 @@
+import { useContext } from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { useStripes } from '@folio/stripes/core';
+import {
+  Accordion,
+  FilterAccordionHeader,
+} from '@folio/stripes/components';
+
+import CheckboxFacet from '../CheckboxFacet';
+import DataContext from '../../contexts/DataContext';
+import { useLocationsOfAllTenantsQuery } from '../../hooks';
+import { getFacetOptions } from '../../facetUtils';
+import { isUserInConsortiumMode } from '../../utils';
+
+const EffectiveLocationFacet = ({
+  closedByDefault,
+  open,
+  selectedOptions,
+  facetOptions,
+  isLoadingFacets,
+  name,
+  onChange,
+  onClear,
+  onFetchFacets,
+  onFilterSearch,
+}) => {
+  const stripes = useStripes();
+  const intl = useIntl();
+  const { allTenantIds } = useContext(DataContext);
+
+  const {
+    data: locationsOfAllTenants,
+    isLoading: isLoadingAllLocations,
+  } = useLocationsOfAllTenantsQuery({ tenantIds: allTenantIds, enabled: open });
```

Comment:
For facets in Inventory app we can't simply send a request to `/locations` and use the response as options because the response doesn't have facet counts.
And it doesn't fix the root cause - `/facets` endpoint returns all locations, but we're only requesting locations from current tenant in `DataProvider` so `getFacetOptions` can't find a match by id.
To fix this we need to use `useLocationsOfAllTenantsQuery` inside `DataProvider` and leave facets as-is.

Reply:
> For facets in Inventory app we can't simply send a request to /locations and use the response as options because the response doesn't have facet counts.

/facets response returns ids, we use those ids to find names of options in /locations response.

> And it doesn't fix the root cause - /facets endpoint returns all locations, but we're only requesting locations from current tenant in DataProvider so getFacetOptions can't find a match by id.

I think the issue is that the locations of other tenants should be shown too.